README.md

```diff
@@ -1 +1,76 @@
-# haaukins-api
\ No newline at end of file
+# Haaukins API
+
+Haaukins API allows the user to connect directly Kali Linux VM avoiding normal Haaukins steps. Unlike Haaukins, in which
+you need to have an event up and running in order to sign up and connect Kali Linux VM, on the API you just need to select
+the challenges you want to have in your Environment and run it. In almost a minute the environment will be ready and the 
+user will be automatically redirect to Guacamole that render the Kali linux VM.
+
+### Keywords used on this repository
+- Client: Identify (through session cookie containing an ID) who made the request on the API
+- Environment: Identify the challenges requested from the Client. It contantains a guacamole instance, a lab and a timer.
+- Challenges: They are Haaukins Exercises
+
+The relation in between those 3 keywords is the following 
+> Client 1 ----> N Environment 1 ----> M Challenges
+
+### API implementation
+The API has some constrains in order to don't create too many Environment and 
+- the API has a Max amount of requests that can handle (specify the value on the config file)
+- a Client has a Max amount of requests that can make
+- the Environment will be closed after 45 minutes it has been created thanks the timer.
```

Comment:
Let's change 45 minutes to something else which is not static, like variable name, I think, this could be more suitable in my opinion. 
- The environment will destroy itself after specified amount of time in source code.